src/main/java/chessmaster/storage/Storage.java

```diff
@@ -254,10 +251,13 @@ public Player loadCurrentPlayer() throws ChessMasterException {
         }
 
         if (fileScanner.hasNext()) {
-            String currentPlayerString = fileScanner.nextLine();
-            ChessBoard board = new ChessBoard(loadPlayerColor(), loadBoard());
-            boolean isCPU = currentPlayerString.equals("CPU");
-            return isCPU ? new CPU(loadPlayerColor().getOppositeColour(), board) : new Human(loadPlayerColor(), board);
+            try {
+                String currentColorString = fileScanner.nextLine();
+                boolean isBlack = currentColorString.equals("black");
```

Comment:
I just realised that if the color was editted to EMPTY, there would be an error in the game so we got to include a check for empty color in `parsePlayerColor` method. (Since the a player color can never be empty)
1. In Color.java, create a `isEmpty` method
2. Check that the converted color is not empty:
```java
try {
    Color color = Color.valueOf(inputColorString);
    if (color.isEmpty()) {
        throws new ParseColorException();
    }
    return color;
} catch (IllegalArgumentException e) {
    throw new ParseColorException();
}
```